### PR TITLE
Add some Storybook stories

### PR DESCRIPTION
**Pull request by dylancom**

### CODE REVIEW

File: src/app/components/LndHub/LndHub.stories.jsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+import LndHub from ".";
```
Comment (bumi) [inline src/app/components/LndHub/LndHub.stories.jsx:3]: what's actually a good convention here: As this is a form should this be called `LndHubForm`

**Review by bumi — APPROVED** [review]
guess this is mergable?

File: src/app/components/LndHub/LndHub.stories.jsx (hunk 1 above)
Comment (dylancom) [reply src/app/components/LndHub/LndHub.stories.jsx:3]: That would be a good convention. And maybe bundle it in a Forms folder for e.g. if there are many.

**Comment by dylancom** [comment]
yes it's mergable, just some small stories.

**Pull request merged**